Add actionable steps and rename ES2015 modules to ECMAScript modules (#3021)
GoogleChrome/web.dev: src/site/content/en/blog/commonjs-larger-bundles/index.md

## Conclusion

**To ensure the bundler can successfully optimize your application, avoid depending on CommonJS modules, and use ECMAScript module syntax in your entire application.**

> Few actionable tips to verify you're on the optimal path:
> - Using Rollup.js' [node-resolve](https://github.com/rollup/plugins/tree/master/packages/node-resolve) plugin you can specify you want to depend only on ECMAScript modules by setting the `modulesOnly` flag
> - To verify if an npm package uses ECMAScript modules you can use the package [`is-esm`](https://github.com/mgechev/is-esm)
> - If you're using Angular, by default you'd get a warning if you depend on non-tree-shakable module

Reviewer: ```suggestion
Here are a few actionable tips to verify you're on the optimal path:

- Use Rollup.js's [node-resolve](https://github.com/rollup/plugins/tree/master/packages/node-resolve)
  plugin and set the `modulesOnly` flag to specify that you want to depend only on ECMAScript modules.
- Use the package [`is-esm`](https://github.com/mgechev/is-esm)
  to verify that an npm package uses ECMAScript modules.
- If you're using Angular, by default you'll get a warning if you depend on non-tree-shakeable modules.
```
